projects/avodata/firstStage.py

```diff
@@ -0,0 +1,121 @@
+import numpy as np
+import math
+
```

Comment:
Please give some general explanation of this file, and the different matrices and their dimensions.
It's basically what you wrote for `find_c_and_a` (now commented out).

Reply:
yes :)